app/assets/javascripts/dropdown.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function(){
```

Comment:
These were inherited from http://bitters.bourbon.io/ , I'm going to convert to coffeescript to keep them consistent with the rest of the codebase.